### ISSUE TITLE
parallel session.get_transactions calls lead to empty transactions array

### ISSUE THREAD

**Issue by Levino** [figo-connect/node-figo#15]
If i query for multiple accounts in parallel, I get empty arrays back. If I query in series it works.

Very bad bug! Who knows how many services will be querying at the same time?

**Comment by JeremyCraigMartinez** [comment]
Can you please paste a code snippet of what you were trying to do?

**Comment by Levino** [comment]
Check this out this branch https://github.com/Levino/figo-to-ynab/tree/parallel-bug-demo. Master works fine. Only difference is this: https://github.com/Levino/figo-to-ynab/commit/7c3f27581b9af95c93ad74d49d4fb2114aa323bd

If you have more than one account, the "master" code will produce files with data for all accounts on ```npm test```. "parallel-bug-demo" branch will produce empty files for all but the first account on ```npm test```. You need to set environment variables to login. Check the Readme: https://github.com/Levino/figo-to-ynab/blob/master/README.md

**Comment by Levino** [comment]
Looks like this has been "fixed" here https://github.com/figo-connect/node-figo/commit/3a58eeaa361c51e365d904f545ed7a1bc9c6b31f I was using 1.3.x and not 1.4.x.

I think this is the worst case to fix this. Very (!!!) un-nody. Of course everyone expects this lib to handle requests in parallel!

**Comment by Levino** [comment]
Closing in favour of #20
